docs/source/install.rst

```diff
@@ -1,97 +1,94 @@
 .. _install:
 
-
 ============
 Installation
 ============
 
-These installation instructions explain how to install the Jupyter Notebook and
-the IPython kernel.
-
-While Jupyter allows users to run code in many different
-programming languages, the Jupyter Notebook itself is implemented in Python.
-To install Jupyter Notebook, you will also need Python installed on your system.
+This section explains how to install the Jupyter Notebook and the IPython
+kernel.
 
-**Installation Scenarios**
+While Jupyter runs code in many different programming languages, Python is a
+prerequisite for installing Jupyter notebook. Select one of the scenarios to
+install Jupyter:
 
-* :ref:`New user: New to Python and Jupyter <new-to-python-and-jupyter>`
-* :ref:`Experienced Python user: New to Jupyter <existing-python-new-jupyter>`
-* :ref:`Upgrading Jupyter <upgrading>`
+    - :ref:`Jupyter (new to Python) <new-to-python-and-jupyter>`
+    - :ref:`Jupyter (experience with Python) <existing-python-new-jupyter>`
+    - :ref:`Upgrading a Jupyter installation <upgrading>`
 
 .. note::
 
-     All commands shown below must be run in the Terminal (Mac/Linux) or Command
-     Prompt (Windows)
+     All installation commands should be run in the Terminal (for Mac and Linux)
+     or the Command Prompt (Windows).
 
 .. _new-to-python-and-jupyter:
 
-New Users: New to Python and Jupyter
-------------------------------------
+Jupyter (new to Python)
+-----------------------
 
-If you're new to Python, we highly recommend `installing Anaconda
-<https://www.continuum.io/downloads>`_. Anaconda includes and conveniently
+For new users, we **highly recommend** `installing Anaconda
+<https://www.continuum.io/downloads>`_. Anaconda conveniently
 installs Python, the Jupyter Notebook, and other commonly used packages for
 scientific computing and data science. Follow Anaconda's instructions for
-downloading the Python 3.5 version.
+downloading and installing the Python 3.5 version.
 
-After you have Anaconda installed, you should make sure that Jupyter is up to
-date. Run the following command in the Terminal (Mac/Linux) or Command
-Prompt (Windows)::
+After installing Anaconda, run the following command in the Terminal or the
+Command Prompt to install Jupyter::
 
     conda install jupyter
 
-Alternatively, you may also use the Anaconda Launcher application to update to
-the latest version.
-
-Now, you may consider :ref:`installing additional kernels <installing-kernels>`
-and :ref:`next steps <next-steps>` such as running the Jupyter Notebook.
+Alternatively, use the Anaconda Launcher application to update to the latest
+Jupyter version.
 
+See :ref:`installing additional kernels <installing-kernels>` to use programming
+languages other than Python and :ref:`next steps <next-steps>` for running the
+Jupyter Notebook.
```

Comment:
Should we switch these two round? I would point people to running it as the default next step.